l10n_eu_oss_oca/__manifest__.py

```diff
@@ -3,19 +3,20 @@
 
 {
     "name": "L10n EU OSS OCA",
-    "version": "15.0.2.0.0",
+    "version": "15.0.2.1.0",
     "category": "Accounting & Finance",
     "website": "https://github.com/OCA/account-fiscal-rule",
     "author": "Sygel Technology," "Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "application": False,
     "installable": True,
     "development_status": "Production/Stable",
-    "depends": ["account", "account_fiscal_position_partner_type"],
+    "depends": ["account", "account_fiscal_position_partner_type", "sale"],
```

Comment:
This new dependency is not OK. You can perform accounting without sales.

Reply:
We added the sales dependency because the same fix would also apply to sales. Do you want us to create a new module for the extension in sales?